lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -301,6 +307,31 @@ func getHEConfig(gwConfig *lte_models.GatewayHeConfig) *lte_mconfig.PipelineD_HE
 		EncryptionKey:          gwConfig.EncryptionKey,
 	}
 }
+
+func getMMEPoolConfigs(networkID string, poolingConfig lte_models.CellularGatewayPoolRecords, cellGateway configurator.NetworkEntity, graph configurator.EntityGraph) (*lte_models.CellularGatewayPoolRecord, uint32, error){
+	// Currently, having multiple (mme group ID, mme code, mme relative
+	// capacity) tuples is unsupported. As such, use the first pool record
+	// to set all of these values.
+	if len(poolingConfig) == 0 {
+		return &lte_models.CellularGatewayPoolRecord{
+			MmeCode:             1,
+			MmeRelativeCapacity: 10,
+		}, 1, nil
```

Comment:
not obvious unless someone reads the whole function implementation what the second returned value of uin32 signify.

Reply:
+1. These 1/0 returns are magic numbers https://flaviocopes.com/magic-numbers/